### DISCUSSION TITLE
The map isn't showing on the production environment.

### DISCUSSION THREAD

**Discussion by Abdul-Rahman-Saleh** [maplibre/maplibre-gl-js#2423] [Q&A]
I'm using maplibre gl js in a react project and it's working properly on the development environment but for some reason map isn't showing on the production env. 

i worte the same code on codesandbox and worked properly

what do you think the problem is??

**Comment by novotnytom** [comment]
Hi! It's common issue if you are using the create-react-app starter... Check these two links: https://github.com/maplibre/maplibre-gl-js/issues/1011 and https://docs.mapbox.com/mapbox-gl-js/guides/install/#targeting-transpilation-to-es6-with-browserslist
